HW_5/task_3.py

```diff
@@ -0,0 +1,24 @@
+# Создайте функцию генератор чисел Фибоначчи (см. Википедию)
+
+
+def fibonacci(num):
+    prefix_fib = '0, 1'
+    start = 1
+    fib_1 = 0
+    fib_2 = 1
+    if num == 0:
+        yield '0'
+    elif num == 1:
+        yield prefix_fib
+    elif num == 2:
+        yield '0, 1, 1'
+    else:
+        yield prefix_fib
+        for i in range(start, num):
+            fib_sum = fib_1 + fib_2
+            fib_1, fib_2 = fib_2, fib_sum
+            yield fib_sum
```

Comment:
Нужно было сделать генератор всех чисел Фибоначчи, а не только `n` первых. В любом случае там кода на 5 строк. Не нужно усложнять. Вот вам код на генерацию `n` первых чисел ряда Фибоначчи. Исправьте его на для генерации всех чисел ряда
```suggestion
def fibonacci(num):
    fib_1, fib_2  = 0, 1
    for _ in range(num):
        yield fib_1
        fib_1, fib_2 = fib_2, fib_1 + fib_2
```